Library/Formula/ansible.rb

```diff
@@ -16,102 +16,269 @@ class Ansible < Formula
   depends_on :python if MacOS.version <= :snow_leopard
   depends_on "libyaml"
 
-  resource "docker-py" do
-    url "https://pypi.python.org/packages/source/d/docker-py/docker-py-0.6.0.tar.gz"
-    sha1 "01eb7b2cd1a607d361170041b973a0e36bb1be42"
+  #
+  # ansible (core dependencies)
+  #
+  resource "Jinja2" do
+    url "https://pypi.python.org/packages/source/J/Jinja2/Jinja2-2.7.3.tar.gz"
+    sha256 "2e24ac5d004db5714976a04ac0e80c6df6e47e98c354cb2c0d82f8879d4f8fdb"
+  end
+
+  resource "MarkupSafe" do
```

Comment:
I kept the original package names from pypi (possibly case sensitive: MarkupSafe, PyYAML,...), but I think that it is fair (and clearer) to apply a "non-case sensitive" alphabetisation. Do you agree?